#重建接口参数
cloud_file_cfg:
  pcd_file: /home/bianjingyang/catkin_realsense/888test_pointcloud_filtered.pcd

reconstruction:
  mesh_resolution: 128  #用去曲面重建
  mesh_resolution_trajectory: 50 #用于重建之后的曲面进行等参数离散化

